source/plugin/permissions.rst

```diff
@@ -153,9 +153,9 @@ Otherwise it will be inherited from any parent ``Subject``. It does not matter w
 
 If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
 default ``Subjects``. Each ``SubjectCollection`` defines its own defaults. The global and weakest default subject can be
-obtained from the ``PermissionService``. It is recommended that plugins define their own permissions to the default's
-transient ``SubjectData`` during every server start-up. This allows the server owner to overwrite the defaults defined
-by plugins according to his needs using the default's persistent ``SubjectData``.
+obtained from the ``PermissionService``. It is recommended (but not mandatory) that plugins define their own permissions
```

Comment:
IMO the defaults are useful for the times where you dont have a custom PermissionService or dont want to configure them. Install it and it works.